packages/wonder-blocks-form/components/choice.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+// This is a 🔘 or ☑️ field item. This component is meant to be used as a child
```

Comment:
Can move this into a comment above the component so that it shows up in Styleguidist.